docs/tutorials/mongoid-queries.txt

```diff
@@ -198,6 +198,195 @@ loaded with ``only`` for those associations to be loaded. For example:
   # => [#<Manager _id: 5c5dc2f0026d7c1730969843, band_ids: [BSON::ObjectId('5c5dc2f0026d7c1730969842')]>]
 
 
+Logical Operations
+******************
+
+Mongoid supports ``and``, ``or``, ``nor`` and ``not`` logical operations on
+``Criteria`` objects. These methods take one or more hash of conditions
+or another ``Criteria`` object as their arguments, with ``not`` additionally
+having an argument-free version.
+
+.. code-block:: ruby
+
+  # and with conditions
+  Band.where(label: 'Trust in Trance').and(name: 'Astral Projection')
+  
+  # or with scope
+  Band.where(label: 'Trust in Trance').or(Band.where(name: 'Astral Projection'))
+  
+  # not with conditions
+  Band.not(label: 'Trust in Trance', name: 'Astral Projection')
+  
+  # argument-less not
+  Band.not.where(label: 'Trust in Trance', name: 'Astral Projection')
+
+For backwards compatibility with earlier Mongoid versions, all of the logical
+operation methods also accept arrays of parameters, which will be flattened
+to obtain the criteria. Passing arrays to logical operations is deprecated and
+may be removed in a future version of Mongoid.
+
+The following calls all produce the same query conditions:
+
+.. code-block:: ruby
+
+  # Condition hashes passed to separate and invocations
+  Band.and(name: 'SUN Project').and(member_count: 2)
+  
+  # Multiple condition hashes in the same and invocation
+  Band.and({name: 'SUN Project'}, {member_count: 2})
```

Comment:
Correct me if I'm wrong here, but it seems like this example is the same as the one below, but with a slightly different syntax. Is that true? Is this also deprecated?

Reply:
Isn't the behavior of these two identical?

Reply:
All of the invocations in this example produce the same result.